lisa/sut_orchestrator/util/device_pool.py

```diff
@@ -0,0 +1,63 @@
+from typing import Any, List, Optional
+
+from lisa.sut_orchestrator.util.schema import HostDevicePoolSchema, HostDevicePoolType
+from lisa.util import LisaException, SkippedException
+
+
+class BaseDevicePoolImpl:
+    def _create_device_pool(
+        self,
+        pool_type: HostDevicePoolType,
+        vendor_id: str,
+        device_id: str,
+    ) -> None:
+        raise NotImplementedError("'_create_device_pool' is not implemented")
+
+    def _get_primary_nic_id(self) -> List[str]:
+        raise NotImplementedError("'_get_primary_nic_id' is not implemented")
+
+    def _get_devices_from_pool(
+        self,
+        pool_type: HostDevicePoolType,
+        count: int,
+    ) -> Any:
+        raise NotImplementedError("'_get_devices_from_pool' is not implemented")
+
+    def _put_devices_into_pool(
```

Comment:
How about `release_devices`?

Reply:
Done

Reply:
@squirrelsc Please check the latest push